### PR TITLE
Paper: example paper for review

### PR DESCRIPTION
**Pull request by deniederhut**
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

*Editor*: @mepa

*Reviewers*: @deniederhut

### CODE REVIEW

**Comment by deniederhut** [comment]
@scoobies assign @mepa as editor

**Comment by scoobies** [comment]
Assigned! @mepa is now the editor

**Comment by deniederhut** [comment]
@scoobies add @deniederhut to reviewers

**Comment by scoobies** [comment]
@deniederhut added to the reviewers list!

File: papers/test_author/author.rst [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+.. class:: keywords
+
+   terraforming, desert, numerical perspective
+
+Introduction
+------------
+
+Twelve hundred years ago  |---| in a galaxy just across the hill...
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum sapien
```
Comment (deniederhut) [inline papers/test_author/author.rst:48]: Hey this looks like this isn't in English -- maybe we could reword this

File: papers/test_author/author.rst [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+       return a + b
+
+With code-highlighting:
+
+.. code-block:: python
+
+   def sum(a, b):
+       """Sum two numbers."""
+      # this is a really long comment, that has enough characters to spill past the column boundary in the two-column paper format
```
Comment (deniederhut) [inline papers/test_author/author.rst:167]: This is a really long comment that is spilling into the next column -- could we wrap this somehow?

**Review by deniederhut — COMMENTED** [review]
Looks good so far! I have some inline suggestions for you to consider.

**Comment by deniederhut** [comment]
@scoobies check references

**Comment by scoobies** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1017/CBO9780511808432 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.5281/zenodo.3509134 is OK
- 10.1038/s41592-019-0686-2 is OK

MISSING DOIs

- 10.25080/majora-92bf1922-00a may be a valid DOI for title: Data Structures for Statistical Computing in Python

INVALID DOIs

- None
```

File: papers/test_author/mybib.bib [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  title     = {pandas-dev/pandas: Pandas},
+  month     = feb,
+  year      = 2020,
+  publisher = {Zenodo},
+  version   = {latest},
+  doi       = {10.5281/zenodo.3509134},
+  url       = {https://doi.org/10.5281/zenodo.3509134},
+}
+
+@InProceedings{pandas2,
```
Comment (deniederhut) [inline papers/test_author/mybib.bib:78]: Hey it looks like this is missing a DOI.

**Comment by deniederhut** [comment]
@scoobies help

**Comment by scoobies** [comment]
Hello @deniederhut, here are the things you can ask me to do:

```

# List all available commands
@scoobies help

# Show our community Code of Conduct and Guidelines
@scoobies code of conduct

# Add to this issue's reviewers list
@scoobies add @username to reviewers

# Remove from this issue's reviewers list
@scoobies remove @username from reviewers

# Assign a user as the editor of this submission
@scoobies assign @username as editor

# Remove the editor assigned to this submission
@scoobies remove editor

# Add a user to this issue's assignees list
@scoobies add assignee: @username

# Remove a user from this issue's assignees list
@scoobies remove assignee: @username

# Builds paper
@scoobies build paper

# Checks build status
@scoobies build status

# Check the references of the paper for missing DOIs
# Optionally, it can be run on a non-default branch 
@scoobies check references
@scoobies check references from branch custom-branch-name

# Label issue with: paper
@scoobies mark paper

# Label issue with: needs-review. Remove labels: unready, needs-more-review, pending-comment, ready
@scoobies mark needs review

# Label issue with: needs-more-review. Remove labels: unready, needs-review, pending-comment, ready
@scoobies mark needs more review

# Label issue with: pending-comment. Remove labels: unready, needs-review, needs-more-review, ready
@scoobies mark pending comment

# Label issue with: ready. Remove labels: unready, needs-review, needs-more-review, pending-comment
@scoobies mark ready

# Label issue with: unready. Remove labels: needs-review, needs-more-review, pending-comment, ready
@scoobies mark not ready

# Label issue with: does_not_build:server
@scoobies mark server fail

# Remove labels: does_not_build:server
@scoobies mark server success

```

**Pull request closed without merging**